pkg/targets/reconciler/twiliotarget/target_adapter.go

```diff
@@ -83,17 +91,24 @@ func makeAppEnv(spec *v1alpha1.TwilioTargetSpec) []corev1.EnvVar {
 
 	if spec.DefaultPhoneFrom != nil {
 		env = append(env, corev1.EnvVar{
-			Name:  "TWILIO_DEFAULT_FROM",
+			Name:  envTwilioDefaultFrom,
 			Value: *spec.DefaultPhoneFrom,
 		})
 	}
 
 	if spec.DefaultPhoneTo != nil {
 		env = append(env, corev1.EnvVar{
-			Name:  "TWILIO_DEFAULT_TO",
+			Name:  envTwilioDefaultTo,
 			Value: *spec.DefaultPhoneTo,
 		})
 	}
 
+	if spec.EventOptions != nil && spec.EventOptions.PayloadPolicy != nil {
+		env = append(env, corev1.EnvVar{
+			Name:  envEventsPayloadPolicy,
+			Value: string(*spec.EventOptions.PayloadPolicy),
+		})
+	}
+
```

Comment:
Can we add BridgeID propagation to the adapter, like you did here?:

https://github.com/triggermesh/triggermesh/blob/6c43c8a267421e36d33de98d0fcf84744a65347b/pkg/targets/reconciler/alibabaosstarget/adapter.go#L89-L92

Reply:
done in https://github.com/triggermesh/triggermesh/pull/78/commits/d7ca4e644923fad01bc0c396c60b330a091df3a3